recipes/conversation_view/feed_pill/feed_item_pill.mdx

```diff
@@ -38,6 +32,12 @@ Can disable the toggleable behaviour (Cursor is reset, no changing of icon on ho
   <Story of={FeedItemPillStories.NoToggleVariant} />
 </Canvas>
 
+## Base Style
+
+<Canvas>
+  <Story of={FeedItemPillStories.Default} />
+</Canvas>
+
```

Comment:
Base style should always be the first example

Reply:
I feel like the control should be next before the variants? otherwise it feels odd to be "controlling" the props and having to scroll to see the changes in the base style.

Reply:
You're not supposed to use controls to see changes on documentation, that's what the playground is for. Having the props there is so you can read abouth props and what they're meant for.